src/components/QuranReader/EndOfScrollingControls/TafsirControls.tsx

```diff
@@ -2,22 +2,36 @@ import React from 'react';
 
 import EndOfScrollingButton from './EndOfScrollingButton';
 
-import { getVerseNavigationUrl, getVerseTafsirNavigationUrl } from 'src/utils/navigation';
+import {
+  getVerseNavigationUrl,
+  getVerseSelectedTafsirNavigationUrl,
+  getVerseTafsirNavigationUrl,
+} from 'src/utils/navigation';
 import { isFirstVerseOfSurah, isLastVerseOfSurah } from 'src/utils/verse';
 import Verse from 'types/Verse';
 
 interface Props {
   lastVerse: Verse;
+  isSelectedTafsir: boolean;
```

Comment:
What does isSelectedTafsir mean? I'm guessing that it has to do with displaying the redux tafsir or route tafsir. We should make this variable name clearer